core/Tracker/Visit/VisitProperties.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Tracker\Visit;
+
+/**
+ * Holds temporary data for tracking requests.
+ *
+ * RequestProcessors
+ */
+class VisitProperties
```

Comment:
> I'm not sure if a plugin developer has to understand the difference between visit and visitor. 

I think it's important to understand the domain when you're coding for it. We should try to educate developers about this. Having real value objects would be better, I think.

>  I'm not happy about VisitProperties as it doesn't sound intuitive to me and really struggled understanding it whenever I saw it but right now I don't come up with a better solution.  I'd personally would probably prefer Visitor even though it's not 100% correct but easier to read but I also understand that someone doesn't want to use it.

Visitor is easier to read, Visit is the easiest ;) I created a somewhat unrelated issue here that contains the creation of value objects for the tracker in the TODO: https://github.com/piwik/piwik/issues/8476

Honestly, I didn't put much thought into the class, just did what I needed. It can be changed.